### DISCUSSION TITLE
Can I Change the font size of the UI?

### DISCUSSION THREAD

**Discussion by alkene5** [johnlindquist/kit#1287] [Q&A] [answered]
After the version update I found that the UI on my computer has become very small, can I adjust the text size ?
<img width="768" alt="image" src="https://github.com/johnlindquist/kit/assets/61105068/7e01b247-34c1-4056-884c-b619753e3882">

**Comment by johnlindquist** [comment, marked as answer]
@alkene0005 `cmd +` and `cmd -` will scale the entire app. `cmd 0` will reset to the defaults.

In your own scripts, you have a lot of control:

```js
let people = ["John Lindquist", "Sarah Drasner", "Chris Biscardi"].map(person => {
  let [name, description] = person.split(" ")
  return {
    name,
    description,
    height: 100,
    nameClassName: "text-2xl",
    descriptionClassName: "text-xl",
  }
})

await mini("Choice Sizes", people)
```
